ELIPs/ELIP-006.md

```diff
@@ -225,11 +223,10 @@ interface IShareManager {
 The flow and code-paths for exiting slashed funds from the protocol have changed. Previously, ERC-20 funds flowed out of the protocol through withdrawals or a burn (transfer to `0x00...00316e4`) at a regular cadence. Native ETH was withdrawn or locked in EigenPods permanently during a slash. Following this upgrade, when a slash occurs, funds are exited in two steps. In order to maintain the protocol guarantee that `slashOperator` should never fail, outflow transfers are non-atomic.
 
 When a single slash occurs...
--Similar to the original burning implementation, burnable shares are first increased in `StrategyManger` storage.
--The `StrategyManager` now interfaces directly with the `SlashEscrowFactory` to setup and initiate the escrow.
--A child `SlashEscrow` contract is then deployed; one is created per `slashId`. These contracts are stateless and immutable.
+-Similar to the original burning implementation, burnable shares are first increased in `StrategyManager` storage.
+-The `StrategyManager` handles the conversion of shares to underlying tokens and their distribution.
```

Comment:
Might want to check the formatting when this is actually rendered -- if you view the file, this does not actually create a line break; the bullets are all on a single line. I _think_ you're just missing a line break before the bullets start?

Reply:
fixed